spec/models/user_spec.rb

```diff
@@ -328,6 +326,14 @@
     end
   end
 
+  describe '#student_team' do
```

Comment:
Since this is a method on the `User` model, it would be cool to also have a scenario for when the user is not a student or has no team. What do you think?